app/pages/search/controls/SearchControlsContainer.js

```diff
@@ -62,7 +62,7 @@ class UnconnectedSearchControlsContainer extends Component {
     const { filters, position } = this.props;
     let hasFilters = Boolean(position);
     ['freeOfCharge', 'end', 'distance', 'duration', 'purpose', 'start', 'unit'].forEach((key) => {
-      if (filters[key]) {
+      if (filters[key] && filters[key] !== '0') {
```

Comment:
Why is this needed?

Reply:
To remove the display of `Reset Filters` button after it is clicked I had to add that line. It seems like the filter value of duration has been provided zero (in number, not in string) and treated at false and affected. So, to fix that I added that code. 
<img width="550" alt="screen shot 2019-03-06 at 10 04 30" src="https://user-images.githubusercontent.com/15159607/53865250-52322200-3ff7-11e9-8a38-ea9faaf77bdd.png">

Reply:
ok. I think we need to improve how search filters are put to the url. I don't think empty filters should be included in the url at all. But that's an entirely different task.

Reply:
We discuss and agree that this condition is not making any change (unnecessary)

Reply:
Now, it is also working without the extra condition. It was not working before. Will investigate a bit and commit after checking if it works correct consistently.